homeassistant/components/zha/light.py

```diff
@@ -175,6 +176,8 @@ def set_level(self, value):
         on at `on_level` Zigbee attribute value, regardless of the last set
         level
         """
+        if self._transitioning:
+            return
```

Comment:
Won't this prevent the level from moving while a transition is happening (so no feedback to the user)? Let's say we have a 60s transition, it is nice to be able to see the slider moving. Could you add a debounce instead of completely discarding updates?

Reply:
No. The problem is when interacting with the slider itself. Watch the videos in the description. If you’re issuing service calls and expecting smooth slider movement I’m not sure what to say...

Reply:
@abmantis wanna try modifying the debounce in light.py to fire at the end of the cool down (`immediate=False`) instead of foreign immediately? Interested to see if it solves the problem at the expense of the delayed update

Reply:
> Won't this prevent the level from moving while a transition is happening (so no feedback to the user)? Let's say we have a 60s transition, it is nice to be able to see the slider moving. Could you add a debounce instead of completely discarding updates?

As far as I know, most light platforms in Home Assistant do not show the brightness slider moving with any transition (see ESPHome or the native Hue integration as an example). Since many Zigbee lights have a default turn on/off transition of 1 second, they also report the intermediate level changes in attribute reports. This is annoying when you just turn on the light and slider jumps around. (IKEA lights actually have a bug that this doesn't happen sometimes)
If you change the slider while the light is already on, the DEFAULT_TRANSITION of 0.1 seconds also makes the slider jump for lights that properly report attribute reports while a transition is happening (like the new Hue lights).

Although I agree that if a transition is longer than (for example) 30 seconds, the slider should perhaps move. I guess the transitioning value could only be set if the transition time is shorter than 30 seconds then. (So attribute reports for long transition times are not ignored. Anything shorter should and will be ignored then.) (So the slider is immediately at the target value and ZHA polls when the light is done transitioning.)

Reply:
@dmulcahey The problem this PR was aiming to solve initially (and that is present on the videos) is related to groups getting updated with states from child devices, right? That only needed the `_ignore_member_changes` part, and not the `_transitioning` restriction, which also applies to non-group entities. 

For groups, I think ignoring member changes during transitions as you did is good. For non-group, I think debouncing the level (with immediate=false) will solve the problem @TheJulianJES described, without killing the transition change in the UI.

And yep, at least for IKEA bulbs the slider slides :) 
This is with a 20sec turn_on transition:
https://user-images.githubusercontent.com/974569/114096381-8b17e180-98b6-11eb-9b2d-bf68fbe63cb4.mp4

Reply:
Both the groups and individual lights suffer from the exact same issue

Reply:
What your video shows is half of it... drag the slider from left to right and watch what happens when you let go

Reply:
Yes, when dragging it jumps around. But (at least on this bulb) a 1 or 2 sec debounce would solve that.

Reply:
So, if I add a 2 second debounce, it will function exactly the same if the reports come in < than 2 seconds apart... each report will cancel the previous one and the end result is the same no?

Reply:
When the user uses the slider, `_transitioning` would not be True so the debouncer logic would not kick in.

Reply:
That’s true if they aren’t overriding the default transition with constant modifier. Or later with the other PR that is in draft. Edge cases are fun 😃. Also, philips and I think IKEA default to a 1 second transition

Reply:
Is there transition when using the slider?

Reply:
If a default is set there will always be a transition